--- chirp-0.2.2/setup.py.inst	2012-03-24 02:01:12.000000000 -0500
+++ chirp-0.2.2/setup.py	2012-06-04 10:39:43.681590064 -0500
@@ -110,7 +110,6 @@
         data_files=[('share/applications', desktop_files),
                     ('share/chirp/images', image_files),
                     ('share/chirp', xsd_files),
-                    ('share/doc/chirp', ['COPYING']),
                     ('share/pixmaps', ['share/chirp.png']),
                     ('share/man/man1', ["share/chirpw.1"]),
                     ('share/chirp/stock_configs', stock_configs),
